core/src/main/java/org/opensearch/sql/analysis/SelectExpressionAnalyzer.java

```diff
@@ -100,6 +109,29 @@ public List<NamedExpression> visitAllFields(AllFields node,
         new ReferenceExpression(entry.getKey(), entry.getValue()))).collect(Collectors.toList());
   }
 
+  @Override
+  public List<NamedExpression> visitNestedAllTupleFields(NestedAllTupleFields node,
+                                                   AnalysisContext context) {
+    TypeEnvironment environment = context.peek();
+    Map<String, ExprType> lookupAllTupleFields =
+        environment.lookupAllTupleFields(Namespace.FIELD_NAME);
+    environment.resolve(new Symbol(Namespace.FIELD_NAME, node.getPath()));
```

Comment:
does this work with aliases?  I think we need to consider resolving the nested symbol with alias instead of path if an alias if given.

Reply:
Consider re-using this for when we want to deal with PartiQL tables.  We will want to have multiple symbol tables for joins, nested, etc.  Being able to generate a 'nested' symbol table from an existing table would be beneficial for PartiQL syntax.

Reply:
Don't think I fully understand what you mean about generating symbol tables for partiql. I see for joins but what is a `nested` symbol table?